src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -111,6 +113,15 @@ public static void MapRequestScrollTo(IScrollViewHandler handler, IScrollView sc
 			if (args is ScrollToRequest request)
 			{
 				var uiScrollView = handler.PlatformView;
+
+				if (uiScrollView.ContentSize == CGSize.Empty && handler is ScrollViewHandler scrollViewHandler)
+				{
+					// If the ContentSize of the UIScrollView has not yet been defined,
+					// we create a pending scroll request that we will launch after performing the Layout and sizing process.
+					scrollViewHandler.PendingScrollToRequest = request;
```

Comment:
This is the main change. Do the scroll to request once we have values in the UIScrollView ContentSize.